subsys/bluetooth/host/l2cap.c

```diff
@@ -1144,6 +1140,7 @@ static void le_conn_req(struct bt_l2cap *l2cap, uint8_t ident,
 	mtu = sys_le16_to_cpu(req->mtu);
 	mps = sys_le16_to_cpu(req->mps);
 	credits = sys_le16_to_cpu(req->credits);
+	result = BT_L2CAP_LE_ERR_PSM_NOT_SUPP;
```

Comment:
This seems like an unnecessary assignment, unless I overlooked something: there is no branch that doesn’t set the value which also ends up using the value.

Reply:
Good catch! Fixed 🙂